functional/platform/nspawn.go

```diff
@@ -395,6 +401,7 @@ UseDNS no
 			return
 		default:
 		}
+		log.Printf("Dialing machine: %s", addr)
```

Comment:
Yes, otherwise we don't have any feedback from tests. btw, we already have `fleetctl list-machines... ` in output. Why don't print `Dialing`?